compiler/coreSyn/CoreLint.hs

```diff
@@ -761,16 +761,11 @@ lintCoreExpr (Let (NonRec tv (Type ty)) body)
 lintCoreExpr (Let (NonRec bndr rhs) body)
   | isId bndr
   = do  { let_ue <- lintSingleBinding NotTopLevel NonRecursive (bndr,rhs)
-        ; let (rhs_ue, update_ue) =
-                case varMult bndr of
-                  Regular w -> (w `scaleUE` let_ue, id)
-                  Alias -> (zeroUE, addAliasUE bndr let_ue)
-        ; (body_ty, body_ue) <-
-            addLoc (BodyOfLetRec [bndr]) $
-            update_ue
-                 (lintBinder LetBind bndr $ \_ -> addGoodJoins [bndr] $ lintCoreExpr body)
-        ; body_ue' <- checkLinearity body_ue bndr
-        ; return (body_ty, body_ue' `addUE` rhs_ue)}
+        ; addLoc (BodyOfLetRec [bndr]) $
+                 (lintBinder LetBind bndr $ \_ ->
+                     addGoodJoins [bndr] $
+                     addAliasUE bndr let_ue $
+                     lintCoreExpr body)}
```

Comment:
Add a link to the note about multiplicity of let-binders.

This diff seems to miss the `let rec` case, where we must force the right-hand sides UE to be scaled by `Omega` (with also a link to the above-mentioned note)